expression/builtin_time_vec_test.go

```diff
@@ -78,6 +78,9 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 	ast.MonthName: {
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETDatetime}},
 	},
+	ast.DayName: {
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETDatetime}},
```

Comment:
Please add more cases to test `vecEvalReal` and `vecEvalInt`.

Reply:
ok, unit test failed, I will fix it